.travis.yml

```diff
@@ -10,12 +10,11 @@ cache:
     - node_modules
 
 before_install:
-  - export PATH=/usr/local/phantomjs-2.0.0/bin:$PATH
-  - "npm config set spin false"
-  - "npm install -g npm@^2"
+  - npm config set spin false
+  - npm install -g bower
+  - npm install phantomjs-prebuilt
 
 install:
-  - npm install -g bower
   - npm install
   - bower install
 
```

Comment:
Indeed it was

2016-04-25 9:31 GMT+02:00 Wojciech Olejnik <notifications@github.com>:

> In .travis.yml
> <https://github.com/Divoolej/linkie-webapp/pull/2#discussion_r60871470>:
>
> >
> >  install:
> > -  - npm install -g bower
> >    - npm install
> >    - bower install
> >
>
> Wait, what - do we use Travis? I didn't know that. We have no tests
> whatsoever at the moment, so.. was this autogenerated or sth?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/Divoolej/linkie-webapp/pull/2/files/f94ebdb4e576cc4e3e42713db033eef116dbaa31#r60871470>
>